src/main/java/pl/niewiemmichal/underhiseye/model/LaboratoryExamination.java

```diff
@@ -0,0 +1,65 @@
+package pl.niewiemmichal.underhiseye.model;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@RequiredArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class LaboratoryExamination {
+
+    @Id @GeneratedValue
+    private Integer id;
+
+    @Lob
+    private String note;
+
+    @Lob
+    private String result;
+
+    @NonNull
+    @Column(nullable = false)
+    private Integer status;
+
+    @Lob
+    private String supervisorNote;
+
+    @NonNull
+    @Column(nullable = false)
+    @Temporal(TemporalType.DATE)
+    private Date orderDate;
+
+    @Column(nullable = false)
+    @Temporal(TemporalType.DATE)
+    private Date completionDate;
+
+    @Column(nullable = false)
+    @Temporal(TemporalType.DATE)
+    private Date approvalDate;
+
+    @NonNull
+    @JoinColumn(nullable = false)
+    @ManyToOne
+    private PhysicalExamination examination;
+
+    @NonNull
+    @JoinColumn(nullable = false)
+    @ManyToOne
+    private LaboratorySupervisor supervisor;
```

Comment:
Ask @ClaudiaPolaczek if this should be nullable or not. Because laboratory supervisor should be able to approve something after it is saved.

Reply:
Sure but approving has nothing to do being assign to a specific task in our system. I guess we should be able to know in advance who is going to review future laboratory examination results.